gossip/src/legacy_contact_info.rs

```diff
@@ -54,16 +54,17 @@ impl Sanitize for LegacyContactInfo {
 #[macro_export]
 macro_rules! socketaddr {
     ($ip:expr, $port:expr) => {
-        std::net::SocketAddr::from((std::net::Ipv4Addr::from($ip), $port))
+        SocketAddr::from((Ipv4Addr::from($ip), $port))
     };
     ($str:expr) => {{
-        $str.parse::<std::net::SocketAddr>().unwrap()
```

Comment:
As CI shows, these absolute paths are needed to use the macros in other contexts. Can you please restore them?

Reply:
Should be fixed.